### PR TITLE
v3.2.1

### PR DESCRIPTION
**Pull request by arbrandes**
Contains two bug fixes.

### CODE REVIEW

File: hastexo/migrations/0005_auto_20190811_1555.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='stacklog',
             name='hook_events',
-            field=jsonfield.fields.JSONField(default=None),
+            field=jsonfield.fields.JSONField(default=b'null'),
```
Comment (fghaas) [inline hastexo/migrations/0005_auto_20190811_1555.py]: I'm a bit puzzled as to why this is a binary string, but I guess this is from `manage.py makemigrations`?

Comment (arbrandes) [reply hastexo/migrations/0005_auto_20190811_1555.py]: Correct.  Straight from `makemigrations`.

**Comment by arbrandes** [comment]
> ~Looks good to me, one minor question.~ Edit: sadly, this breaks on Python 3.

Yeah, nice catch on the binary string.  That's what fails on Python 3.  Fixed the migration manually, once more.

**Comment by arbrandes** [comment]
Wat.  I tested py27 locally and it worked just fine.

**Pull request merged**